src/Microsoft.Extensions.FileSystemGlobbing/PatternMatchingResult.cs

```diff
@@ -15,13 +16,30 @@ public class PatternMatchingResult
         /// </summary>
         /// <param name="files">A collection of <see cref="FilePatternMatch" /></param>
         public PatternMatchingResult(IEnumerable<FilePatternMatch> files)
+            : this(files, hasMatches: files.Any())
```

Comment:
None of our code calls this ctor anymore. Would be unnecessary to make  the code messier when we don't have a good reason to.